### ISSUE TITLE
Stop Keyboard from getting smaller ?

### ISSUE THREAD

**Issue by RickMason1** [Faylixe/pygame-vkeyboard#19]
I made a few mods to one of your files "textinput.py" .  I need to call the keyboard from an existing script multiple times.
The problem is that when I make subsequent calls the keyboard shrinks.  When I do need the keyboard it's just to input 4 or 5 characters.

Your modified script is below with changes that I made surrounded by ####.  I hope it arrives in readable form

Thanx
Rick Mason
[textinput.txt](https://github.com/Faylixe/pygame-vkeyboard/files/5126662/textinput.txt)

**Comment by RickMason1** [comment]
I made a few mods to one of your files "textinput.py" . I need to call the keyboard from an existing script multiple times.
The problem is that when I make subsequent calls the keyboard shrinks. When I do need the keyboard it's just to input 4 or 5 characters.

Your modified script is below with changes that I made surrounded by ####. I hope it arrives in readable form

Thanx
Rick Mason

[textinput.txt](https://github.com/Faylixe/pygame-vkeyboard/files/5127280/textinput.txt)
